torch/onnx/_internal/exporter/_core.py

```diff
@@ -557,10 +596,14 @@ def _add_nodes(
                         constant_farm,
                         registry=registry,
                         opset=opset,
+                        local_functions=local_functions,
                     )
                 else:
                     # No lowering
-                    _handle_call_function_node(model.graph, node, node_name_to_values)
+                    _handle_call_function_node(model, node, node_name_to_values)
+            elif node.op == "get_attr":
+                if hasattr(node, "graph"):
+                    local_functions[node.target] = node.graph
```

Comment:
Is it possible that node.target can be a local (scoped) name? That could complicate this translation. Are we guaranteed that this name will be unique in the entire translation?